### DISCUSSION TITLE
Trying to train from scratch

### DISCUSSION THREAD

**Discussion by arvindksh81** [xinntao/Real-ESRGAN#85] [Q&A] [answered]
Hi,

I am trying to train this network from scratch on my own image data but I don't see convergence in training.
Some details:
1. I am using batch size of 6 with 2 GPUs so total 12 batch size
2. crop size of 256
3. I am initially trying to train with pretrained Real-ESRNet
4. other hyper parameters are same as original

My discriminator loss is hovering around 0.25 after 180K iterations and decreasing continuously and generator gan loss is increasing continously from 0.11 to 0.25.

Is it possible to train this network with 12 batch size ? I could only afford this due to limited size of my compute (2 16GB T4 cards). Please suggest if you see any possibility to train on my GPU machine.

Best Regards,
Arvind

**Comment by xinntao** [comment, marked as answer]
1. The loss of G and D will oscillate, which is normal. Did you check whether the visual quality is improving during training? 
2. For a smaller batch size, you may need to decrease the learning rate.

**Comment by arvindksh81** [comment]
Thanks for your answers Xinntao. I am trying some of these things to make it work.

How long do ESRNet and ESRGAN take to train from scratch on your V100 GPUs ?

**Comment by xinntao** [comment]
@arvindksh81 It is about a week for ESRNet and 3~4 days for ESRGAN. 
You can fine-tune to shorten the training period.

**Reply by arvindksh81** [reply]
Do you think Cut mix data augmentation would be worth exploring on this framework for improving accuracy further?

**Comment by schirrmacher** [comment]
> You can fine-tune to shorten the training period.

How many iterations do you recommend for fine-tuning and how many images typically?
